src/components/Tabs/TabItem.tsx

```diff
@@ -30,7 +29,7 @@ export const TabItem = ({ active, disabled, children, id }: TabItemProps & { act
             ref={ref}
             id={`${id}-content`}
             aria-labelledby={id}
-            className={merge([!active || disabled ? 'tw-hidden' : '', !hasInteractiveElements && FOCUS_VISIBLE_STYLE])}
+            className={merge([!active || disabled ? 'tw-hidden' : '', 'focus:tw-ring-4 focus:tw-ring-blue'])}
```

Comment:
Should we include `tw-rounded` here to ensure the ring style is consistent?

Reply:
Hello, just checked with Johanna, For this case as the component has a square shape, the focus shouldn't be rounded.

Reply:
Is it rounded option missing on the focus ring?